app/settings.py

```diff
@@ -0,0 +1,44 @@
+import os
+
+
+def get_key(key_name):
+    key = open(key_name, 'r')
+    contents = key.read()
+    return contents
+
+LOGGING_LEVEL = os.getenv("LOGGING_LEVEL", "DEBUG")
+
+RABBIT_URL = 'amqp://{user}:{password}@{hostname}:{port}/{vhost}'.format(
+    hostname=os.getenv('RABBITMQ_HOST', 'localhost'),
```

Comment:
These are needed by the unit tests, so removing them breaks the tests.

We need to convert this to a flask app in order to have a health check. Once we do that we can use flask config and remove this settings file completely, hence removing the default values